Extensions/AzureBlob/README.md

```diff
@@ -6,7 +6,10 @@ The Azure Blob Storage extension provides reading and writing of formatted files
 
 ## Settings
 
-Source and Sink settings require the parameters shown below. 
+Source and sink require settings used to locate and access the Azure Blob Storage account. This can be done in one of two ways:
+
+- Using a `ConnectionString` that includes an AccountEndpoint and AccountKey
+- Using RBAC (Role Based Access Control) by setting `UseRbacAuth` to true and specifying `AccountEndpoint` and optionally `EnableInteractiveCredentials` to prompt the user to log in to Azure if default credentials are not available.
```

Comment:
See my earlier comment on this defaulting EnableInteractiveCredentials to true. Not having it default as true is consistent with the description here.

Reply:
default of true has been removed. documentation is now correct.